test/metabase/lib/expression_test.cljc

```diff
@@ -481,20 +482,19 @@
       :filter      [:= [:field 1 {:base-type :type/Integer}] 3])))
 
 (deftest ^:parallel diagnose-expression-test-2
-  (testing "correct expression are accepted silently"
-    (testing "type errors are reported"
-      (are [mode expr] (=? {:message #"Type error: .*"}
-                           (lib.expression/diagnose-expression
-                            (lib.tu/venues-query) 0 mode
-                            (lib.convert/->pMBQL expr)
-                            #?(:clj nil :cljs js/undefined)))
-        :expression  [:/ [:field 1 {:base-type :type/Address}] 100]
-             ;; To make this test case work, the aggregation schema has to be
-             ;; tighter and not allow anything. That's a bigger piece of work,
-             ;; because it makes expressions and aggregations mutually recursive
-             ;; or requires a large amount of duplication.
-        #_#_:aggregation [:sum [:is-empty [:field 1 {:base-type :type/Boolean}]]]
-        :filter      [:sum [:field 1 {:base-type :type/Integer}]]))))
+  (testing "type errors are reported"
+    (are [mode expr] (=? {:message #"Type error: .*"}
+                         (lib.expression/diagnose-expression
+                          (lib.tu/venues-query) 0 mode
+                          (lib.convert/->pMBQL expr)
+                          #?(:clj nil :cljs js/undefined)))
+      :expression  [:/ [:field 1 {:base-type :type/Address}] 100]
+      ;; To make this test case work, the aggregation schema has to be
+      ;; tighter and not allow anything. That's a bigger piece of work,
+      ;; because it makes expressions and aggregations mutually recursive
+      ;; or requires a large amount of duplication.
+      #_#_:aggregation [:sum [:is-empty [:field 1 {:base-type :type/Boolean}]]]
```

Comment:
Why is this line commented out?

Reply:
Good question. I assume because of what the comment just above says. The diff probably doesn't help here, but this line didn't actually change, just the indentation because I removed the `(testing "correct expression are accepted silently"` one the first line of the test.

Reply:
Aside: Github's "Hide whitespace" setting can be helpful to remove indentation-only changes from the diff.